src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -57,6 +57,56 @@ the `ConfigureServices` of your `Startup` class. Refer to documentation for
 For an ASP.NET application, adding instrumentation is typically done in the
 `Global.asax.cs`. Refer to documentation for [OpenTelemetry.Instrumentation.AspNet](../OpenTelemetry.Instrumentation.AspNet/README.md).
 
+## Advanced configuration
+
+This instrumentation can be configured to change the default behavior by using
+`GrpcClientInstrumentationOptions`.
+
+### SuppressDownstreamInstrumentation
+
+This option prevents downstream instrumentation from being invoked.
+Grpc.Net.Client is built on top of HttpClient. When instrumentation for both
+libraries is enabled, `SuppressDownstreamInstrumentation` prevents
+the HttpClient instrumentation from generating an additional activity.
+Additionally, since HttpClient instrumentation is normally responsible for
+propagating context (e.g., W3C trace context and baggage), Grpc.Net.Client
+instrumentation propagates context when `SuppressDownstreamInstrumentation` is
+enabled.
```

Comment:
Thoughts on making the original example above to include AddHttpClient as well? That'll ensure propagation occurs correctly. They may see double span, and if that is an issue, they can leverage supress option then.

Reply:
👍 good call.